docs/preview/features/secret-store/index.md

```diff
@@ -12,16 +12,23 @@ Once register, you can fetch all secrets by using `ISecretProvider` which will g
 
 > :bulb: See [this section](#using-secret-store-within-azure-functions) if you want to use the secret store functionality whitin Azure Functions.
 
-## Using secret store within .NET Core (web, worker...) project
+## Built-in secret providers
```

Comment:
I've moved this up so that we first show what we support rather than how to use it without knowing if you'll use it